lib/plugins/chainableStatus.js

```diff
@@ -0,0 +1,61 @@
+/**
```

Comment:
This software is packaged and distributed as a whole. I'm not sure we want to go down the path of separate license claims for individual contributions.

I am not a lawyer though, I've just never seen a license preamble in a source file that didn't match the projects. Not sure what the implications are.

Reply:
I think this can be resolved by replacing the license by a link to:
https://github.com/restify/node-restify/blob/master/LICENSE